### DISCUSSION TITLE
bug de taille Météo France

### DISCUSSION THREAD

**Discussion by SeddikRebbaj** [witsa/synapps#683] [Q&A] [answered]
Bonjour, je viens de me rendre compte d'un bug d'affichage j'ai fait apparaitre la température de la ville de mon site grâce à un script météo France mais quand je zoom on diminue la taille de l'écran il s'agrandit. Je ne sais pas sur quel paramètre je dois intervenir pour que mon widget météo France s'adapte en fonction de la taille .
Script: <iframe id="widget_autocomplete_preview" width="270vw" height="200vh" frameborder="0" src="https://meteofrance.com/widget/prevision/091940##000000"> </iframe>
![Bug Météo France 2](https://github.com/witsa/synapps/assets/151936804/9b43de4a-d3f6-4af7-8b81-795524fc7dbf)
![Bug Météo France](https://github.com/witsa/synapps/assets/151936804/55eb2ebc-efd0-4369-bec3-2aa9459e546f)

**Comment by WITNicolas** [comment, marked as answer]
Bonjour,
Le plus simple serait de mettre votre iFrame dans un acteur de type BOITE A VUE qui permettra le redimensionnement automatique.

Documentation relative
https://witsa.github.io/synapps/concepts/actor-types/layout-view-box.html#boite-%C3%A0-vue

**Comment by manufitoussiwit** [comment]
Bonjour, 

Ce n'est pas un bug :-)
En html, les unités misent pour la taille de l'iframe `width="270vw" height="200vh"` sont ignorées. Le navigateur considère que c'est des pixels. Donc, il est normal que l'iframe grossisse si on zoom. C'est comme si c'était écrit : `width="270" height="200"`

Le mieux est de suivre le conseil de @WITNicolas ;-)

**Comment by SeddikRebbaj** [comment]
merci beaucoup ça fonctionne

**Comment by cinematographes** [comment]
Bonjour petite question comment à tu peut empilait des images avec les textes par-dessus ?

**Reply by JeremieDENAEGEL** [reply]
Bonjour,
Je pense que son fond de plan a été lié à la propriété "image de fond" de son acteur Toile ou à un acteur empilement (placé en enfant de son acteur Toile)
